### ISSUE TITLE
Edit notes aren't working

### ISSUE THREAD

**Issue by Kewbey** [ppy/osu-web#1514]
There is no possibility to add edit notes to the posts while editing them, also old edit notes from old forums aren't shown at all.

**Comment by peppy** [comment]
Are these actually worth having?

**Comment by cl8n** [comment]
I think so, because they clarify which part of the post was changed by a moderator (rather than the OP)

**Comment by Kewbey** [comment]
Yes, those are pretty useful for moderators for noting what was edited in the post, if it wasn't deleted completely. Or mention the changes in some local language forums. They were even used in management forums for mention changes in some information threads, or by QATs when they react to reported maps.

**Comment by nanaya** [comment]
Can't it be written at the end of the post content?

**Comment by Nath789** [comment]
Everytime you edit a post, "Last edited by..." message will be visible to public. I think the past behavior is better for reasons where if you didn't wrote any notes, it won't appear nor update the edit note. So I can also say it is worth having.

**Comment by Kewbey** [comment]
End of the post content doesn't sound appealing here, because I think there should be a distinguishable appearance between simple post content and moderator notes, so users won't be able to edit their posts making them look like a moderator edited it. Also, as Nathanael said, those notes also were adding the possibility to edit the posts without leaving the info about who and when edited it, it's also an useful feature just for same reasons why the info about who silenced the user isn't public.
